Babylon authoritarian general changes

Unintuition-Time opened the pull request:
Pursuant to #95 , changes to general approval and law enactment approval, and authority. Ready for review @HiddeLekanne @Danarca

Unintuition-Time committed:
Added Secession, IG Approval Setup

Unintuition-Time committed:
Changed Approval Multipliers for lawgroups

Unintuition-Time committed:
Change IG approvals min/max

Unintuition-Time committed:
Changed Legitimacy

Unintuition-Time committed:
Modified Radicals/Loyalists from Legitimacy

to keep inline with the legitimacy level changes

Unintuition-Time committed:
Removed Legitimacy Changes

Not an appropriate time to implement them

Unintuition-Time committed:
Final Changes

for Authoritarian general changes. Balance Patch Changes are pending

Unintuition-Time committed:
Forgot this

Authority deficit penalty

Unintuition-Time committed:
Update babylon_defines.txt

HiddeLekanne commented on common/law_groups/00_babylon_law_groups.txt:
```diff
@@ -1,7 +1,7 @@
 ﻿lawgroup_diplomatic_stance = {
 	base_enactment_days = 180
```
It can also greatly be sped up like maybe 30 days or something. I was planning to do something like that for tweaks.

Unintuition-Time replied:
Fixed, ready for second review @HiddeLekanne

Unintuition-Time committed:
lowered enactment time of diplo stance

HiddeLekanne commented on common/defines/babylon_defines.txt:
```diff
@@ -93,3 +93,31 @@ NPops = {
 	MASS_MIGRATION_FEW_TARGET_STATES_PENALTY = 0.2	#0.25	# Mass migration target score is divided by ( 1 + this * number of possible target states less than MASS_MIGRATION_NUM_TARGET_STATES_FOR_NO_PENALTY )
 }
+
+NPolitics = {
```
I would like to keep the original values in the comments like always.

HiddeLekanne commented on common/law_groups/00_laws.txt:
```diff
@@ -0,0 +1,200 @@
+	}
+}
+
+lawgroup_distribution_of_power = {
```
Maybe also here keep the original values in comments, I don't know how much work this is. But specifically for enactment)days and approval, it would be nice to have the vanilla values in there.

Unintuition-Time replied:
I'll update both sections with those before asking for a second review, yea. I think I just forgot to do it while implementing all the changes.

Unintuition-Time committed:
Added original values

added for each of the new values in defines and law groups

HiddeLekanne commented on common/defines/babylon_defines.txt:
```diff
@@ -96,10 +96,10 @@ NPops = {
-	MAX_IG_APPROVAL = 25				# Maximum IG approval
-	MIN_IG_APPROVAL_FROM_LAWS = -8		# IG cannot have approval lower than this as a total from laws 'on the book'
-	MAX_IG_APPROVAL_FROM_LAWS = 8		# IG cannot have approval higher than this as a total from laws 'on the book'
+	MIN_IG_APPROVAL = -25				#20 Minimum IG approval
+	MAX_IG_APPROVAL = 25				# 20 Maximum IG approval
```
What is the point of this? The thresholds don't use this extra opinion, so why would 5 extra approval or -5 extra approval matter?

HiddeLekanne commented on common/law_groups/00_laws.txt:
```diff
@@ -25,15 +25,15 @@ lawgroup_governance_principles = {
 	law_group_category = power_structure
 	base_enactment_days = 150
-	enactment_approval_mult = 1.2
+	enactment_approval_mult = 1.2 #1
```
In combination with the defines changes this means that the approval impact is 7 instead of 8 for this lawgroup, is this on purpose?

HiddeLekanne commented on common/law_groups/00_laws.txt:
```diff
@@ -25,15 +25,15 @@ lawgroup_governance_principles = {
 lawgroup_citizenship = {
 	law_group_category = power_structure
-	enactment_approval_mult = 1.2
+	enactment_approval_mult = 1.2 #1
```
In combination with the defines changes this means that the approval impact is 7 instead of 8 for this lawgroup, is this on purpose?

Unintuition-Time committed:
Updated Min and Max Approval

This will show players the exact and real amount their current approval with an IG sits at, for better or worse.
